VirtualBox.spec

```diff
@@ -398,7 +398,6 @@ kmk %{_smp_mflags}    \
 %{?with_docs:   VBOX_WITH_DOCS=1 }                             \
     VBOX_JAVA_HOME=%{_prefix}/lib/jvm/java  \
     VBOX_WITH_UPDATE_REQUEST=               \
-    VBOX_WITH_VIRTIO=                       \
```

Comment:
Why this option was dropped without any comment ?

Please add comments and forward to bugzilla.rpmfusion.org

Reply:
It's detailed in https://bugzilla.rpmfusion.org/show_bug.cgi?id=5747